hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -318,7 +369,8 @@ private StorageDescriptor storageDescriptor(TableMetadata metadata, boolean hive
     return storageDescriptor;
   }
 
-  private long acquireLock() throws UnknownHostException, TException, InterruptedException {
+  // Visible for testing
```

Comment:
nit: Could we use
`org.apache.iceberg.relocated.com.google.common.annotations.VisibleForTesting`?